e2e/bitrise.yml

```diff
@@ -0,0 +1,195 @@
+    - XCODE_SIMULATOR_OS_VERSION: "latest"
+    - XCODE_SIMULATOR_PLATFORM: iOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+    - _check_exported_artifacts
+    - _check_xcpretty_html_report
+
+  test_failing_ui_test:
+    steps:
+    - script:
+        title: Start a failing workflow, wrapped in a script.
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -x # Do not set -e as bitrise command is excepted to fail
+            bitrise run --config=./e2e/bitrise.yml utility_objc_failing_ui_test
+            if [ $? -ne 1 ] ; then
+              echo "Workflow was excepted to fail, exit code not 1."
+              exit 1
+            fi
+
+  utility_objc_failing_ui_test:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
+    - TEST_APP_BRANCH: failing-ui-test
+    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
+    - BITRISE_SCHEME: ios-simple-objc
+    - XCODE_SIMULATOR_DEVICE: iPhone 8 Plus
+    - XCODE_SIMULATOR_OS_VERSION: "latest"
+    - XCODE_SIMULATOR_PLATFORM: iOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+    - _check_exported_artifacts
+
+  test_tvos:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-apps-tvos-swift.git
+    - TEST_APP_BRANCH: master
+    - BITRISE_PROJECT_PATH: NPO Live.xcworkspace
+    - BITRISE_SCHEME: NPO Live
+    - XCODE_SIMULATOR_DEVICE: Apple TV
+    - XCODE_SIMULATOR_OS_VERSION: "latest"
+    - XCODE_SIMULATOR_PLATFORM: tvOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+
+  test_parallel:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
+    - TEST_APP_BRANCH: master
+    - BITRISE_PROJECT_PATH: BullsEye.xcodeproj
+    - BITRISE_SCHEME: BullsEye
+    - XCODE_SIMULATOR_DEVICE: iPhone 8 Plus
+    - XCODE_SIMULATOR_OS_VERSION: "latest"
+    - XCODE_SIMULATOR_PLATFORM: iOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+
+  _run:
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -ex
+            rm -rf ./_tmp
+    - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git:
+        inputs:
+          - repository_url: $TEST_APP_URL
+          - clone_into_dir: ./_tmp
+          - branch: $TEST_APP_BRANCH
+    - certificate-and-profile-installer: {}
+    - path::./:
+        inputs:
+        - project_path: ./_tmp/$BITRISE_PROJECT_PATH
+        - scheme: $BITRISE_SCHEME
+        - output_tool: $XCODE_OUTPUT_TOOL
+        - simulator_device: $XCODE_SIMULATOR_DEVICE
+        - simulator_os_version: $XCODE_SIMULATOR_OS_VERSION
+        - simulator_platform: $XCODE_SIMULATOR_PLATFORM
+        - is_clean_build: "yes"
+        - should_build_before_test: "yes"
+        - xcodebuild_test_options: -verbose
+        - export_uitest_artifacts: "true"
+        - verbose: "yes"
+    - change-workdir:
```

Comment:
with this workflow we do not leave the ORIG_BITRISE_SOURCE_DIR, no need for this step